hack/verify-api-groups.sh

```diff
@@ -111,12 +111,10 @@ for expected_install_package in "${expected_install_packages[@]}"; do
 		continue
 	fi
 
-	for known_version_file in "${known_version_files[@]}"; do
-		if ! grep -q "${expected_install_package}/install" ${known_version_files} ; then
-			echo "missing ${expected_install_package}/install from ${known_version_files}"
-			exit 1
-		fi
-	done
+	if ! grep -q "${expected_install_package}/install" "${known_version_files[@]}" ; then
```

Comment:
is this actually equivalent? the original code had a bug on line 115 where it should have been ${known_version_file}, we fixed this where it was copied in other scripts

Reply:
I think that the original code was intended to execute the following command.
So, I changed it to ${known_version_files[@]}.
```
grep -q "${expected_install_package}/install" pkg/master/import_known_versions.go pkg/client/clientset_generated/internal_clientset/import_known_versions.go
```

"known_version_files" contains the following files:
  - pkg/master/import_known_versions.go
  - pkg/client/clientset_generated/internal_clientset/import_known_versions.go  <= this file is not exist in my environment.

If it is changed to ${known_version_file}, this script will exit with "No such file or directory".
Should I change to ${known_version_file} and remove a non-existent file from known_version_files?

Reply:
> Should I change to ${known_version_file} and remove a non-existent file from known_version_files?

yes, we've done that one elsewhere.